framework/widget/dtd/widget-form.xsd

```diff
@@ -857,7 +857,13 @@ under the License.
                     </xs:documentation>
                 </xs:annotation>
             </xs:attribute>
-            <xs:attribute name="disabled" type="xs:boolean" default="false"/>
+            <xs:attribute type="xs:string" name="disabled">
+                <xs:annotation>
+                    <xs:documentation>
+                        Set a field as disabled : not mutable, not focusable, not submitted if in a form.
+                    </xs:documentation>
```

Comment:
Could be worth including something about using expressions, similar to that the for `id-name` attribute:
```
Accepts ${} notation to allow use of expressions. Expressions must evaluate to the strings 'true', 'false' (without quotes) or empty (equivalent to false)
```

Reply:
done